src/protocols/rdp/Makefile.am

```diff
@@ -233,6 +233,7 @@ rdp_keymaps =                                \
     $(srcdir)/keymaps/en_gb_qwerty.keymap    \
     $(srcdir)/keymaps/en_us_qwerty.keymap    \
     $(srcdir)/keymaps/es_es_qwerty.keymap    \
+    $(srcdir)/keymaps/es_es_dvorak.keymap    \
```

Comment:
As with the client-side changes, the `es_es_dvorak` entry should go above the `es_es_qwerty` entry...

Reply:
Done. I'll be checking build failure.